docs/dev/swagger.yaml

```diff
@@ -65,6 +65,10 @@ definitions:
         $ref: '#/definitions/LocaleObject'
       discipline:
         $ref: '#/definitions/LocaleObject'
+      agency:
+        $ref: '#/definitions/LocaleObject'
+      keyword:
```

Comment:
Update the property names here as was mentioned this comment: https://github.com/pkp/pkp-lib/pull/4999#discussion_r314236644